shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -942,10 +942,10 @@ - (FlutterEngine*)spawnWithEntrypoint:(/*nullable*/ NSString*)entrypoint
   SetEntryPoint(&settings, entrypoint, libraryURI);
 
   flutter::Shell::CreateCallback<flutter::PlatformView> on_create_platform_view =
-      [self](flutter::Shell& shell) {
-        [self recreatePlatformViewController];
+      [result](flutter::Shell& shell) {
+        [result recreatePlatformViewController];
         return std::make_unique<flutter::PlatformViewIOS>(
-            shell, self->_renderingApi, self->_platformViewsController, shell.GetTaskRunners());
```

Comment:
This was the root of all the problems I experienced the last couple of days.  I don't think we need the ScopedTaskRunner.  I got confused..

Reply:
> This was the root of all the problems I experienced the last couple of days. I don't think we need the ScopedTaskRunner.

Wait rly? Ha, awesome.

Reply:
Yea, so I was getting crashes inside the raster thread after the spawner was killed.  It was a crash talking to the external view embedder.  So, it appeared that the raster event queue had a queued event talking to that deleted rasterizer.  It turns out the rasterizer that was involved in the crash was actually the spawned engines rasterizer.  So, the event was correctly going forward with a live rasterizer but it ultimately was referencing something that had been deleted, the platformViewsController.  It's easy to overlook that that is a shared_ptr, even then I didn't expect there to be a crash inside.  The shared_ptr is kind of pointless if it doesn't actually maintain ownership of its children (or use weak ptrs).